pkg/client/k8s/k8s.go

```diff
@@ -87,6 +87,9 @@ const (
 	// StroagePoolClaimCRKK is a K8s CR of kind StoragePool
 	StroagePoolClaimCRKK K8sKind = "StoragePoolClaim"
 
+	// CStorPoolClusterCRKK is a K8s CR of kind CStorPoolCluster
```

Comment:
Changes to this file k8s/k8s.go may not be required. Please refer to my next comments.